auth0/README.md

```diff
@@ -2,7 +2,33 @@
 
 ## Overview
 
-Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+Auth0, the identity platform for development teams, provides developers and enterprises with the building blocks they need to secure their applications.
+
+
+### Integration overview
+
+The integration with Datadog is enabled by Auth0’s Log Streaming. This capability works by sending batches of log events as they are generated in Auth0, giving customers up-to-date information about their Auth0 tenant. Log streaming is capable of delivering 10X more logs, and guarantees delivery with error handling. Today we are also releasing a generic webhook allowing you to deliver near real-time logs to most third-party tools.
+
+
+#### The integration with Datadog provides a number of important benefits:
+
+Visualize Auth0 Data Without Extra Development Time
+One of the key benefits of using Datadog is the ability to collect and visualize data in order to identify trends. Engineering teams use it to visualize error rates and traffic data. Security teams use it to visualize authorization traffic and set up alerts for high-risk actions.
+
+
+#### Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
```

Comment:
```suggestion
#### Identity data

Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
```

This is a pretty long header. Might be better to have a have a shortened header here and then have this be the lead-in sentence for this section.